schema/article.js

```diff
@@ -18,9 +18,18 @@ const ArticleType = new GraphQLObjectType({
     title: {
       type: GraphQLString,
     },
+    published_at: {
+      type: GraphQLString,
+    },
+    updated_at: {
+      type: GraphQLString,
+    },
     thumbnail_title: {
       type: GraphQLString,
     },
+    thumbnail_teaser: {
+      type: GraphQLString,
+    },
```

Comment:
it's the blurb preview that gets used in this case in the /artist/articles section... 
![screen shot 2016-05-24 at 12 00 42 pm](https://cloud.githubusercontent.com/assets/3171662/15500046/4a02ee1e-21a7-11e6-9115-e1c41b233995.png)
![screen shot 2016-05-24 at 12 01 37 pm](https://cloud.githubusercontent.com/assets/3171662/15500047/4a1b8ff0-21a7-11e6-9873-d41cc9fc30e8.png)